.github/ISSUE_TEMPLATE.md

```diff
@@ -1,27 +1,4 @@
-<!--
-THIS IS NOT A HELP FORUM.
-If you are experiencing problems with setting up Jest, please make sure to visit our Help page:
-https://facebook.github.io/jest/help.html
--->
+## 👉 [PLEASE CHOOSE AN ISSUE TEMPLATE HERE](https://github.com/facebook/jest/issues/new/choose) 👈
 
-<!--
-Before creating an issue please check the following:
-* you are using the latest version of Jest
-* try re-installing your node_modules folder
-* run Jest once with `--no-cache` to see if that fixes the problem you are experiencing.
--->
-
-**Do you want to request a _feature_ or report a _bug_?**
-
-**What is the current behavior?**
-
-**If the current behavior is a bug, please provide the steps to reproduce and
-either a repl.it demo through https://repl.it/languages/jest or a minimal
-repository on GitHub that we can `yarn install` and `yarn test`.**
-
-**What is the expected behavior?**
-
-**Please provide your exact Jest configuration**
-
-**Run `npx envinfo --preset jest` in your project directory and paste the
-results here**
+This ticket will be immediately closed for not following one of the above issue
```

Comment:
"if not following" instead of "for not following"?

Reply:
Updated